tests/liberror/errors.go

```diff
@@ -0,0 +1,3 @@
+package liberror
+
+var CouldNotFindComputeContainer = "could not find compute container for pod"
```

Comment:
Errors usually belong to some owner. Here it seems like the owner is "tests", which does not make sense (as it does not expose errors).

Can you find a proper owner to it?
As a self contained package it makes no sense IMO.

Reply:
Moving it to libvmi, since it's an error looking for virt-launcher compute container.

Reply:
`libvmi` is about building vmi specs. It does not do anything else at the moment.

Reply:
Maybe it fits your new `libpod`.